MIPs/mip-x-authorize.md

```diff
@@ -0,0 +1,294 @@
+          "nativeCurrency": {
+              "name": "ETH"
+              "symbol": "ETH"
+              "decimals": 18
+          },
+          "iconURLs": ["https://example.com/linea.svg"] 
+        }],
+      },
+    },
+  },
+}
+```
+
+### Response
+An example of the corresponding JSON-RPC response that an application would receive from the wallet:
+
+```js
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": {
+    "sessionScopes": { 
+      "wallet": {
+        "methods": ["wallet_scanQRCode"],
+        "notifications": ["walletAuthorizationChanged"]
+      }   
+      "eip155:wallet": {
+          "methods": ["eth_signTypedData_v4","personal_sign","wallet_addEthereumChain"],
+          "accounts": ["0x0910e12C68d02B561a34569E1367c9AAb42bd810"]
+      },
+      "eip155:1": {
+        "methods": ["eth_sendTransaction", "eth_getBalance","eth_blockNumber","eth_getTransactionCount","wallet_watchAsset"],
+        "notifications": ["message"],
+        "accounts": ["eip155:1:0x0910e12C68d02B561a34569E1367c9AAb42bd810"]
+      },
+      "eip155:59144": {
+        "methods": ["eth_sendTransaction", "eth_getBalance","eth_blockNumber","eth_getTransactionCount","wallet_watchAsset"],
+        "notifications": ["message"],
+        "accounts": ["eip155:59144:0x0910e12C68d02B561a34569E1367c9AAb42bd810"]
+      },
+    },      
+  }
+}
+```
+
+# Proposal
+
+## Language
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" written in uppercase in this document are to be interpreted as described in RFC 2119.
+
+## Definitions
+
+**network**: Decentralized networks where control, validation, and decision-making processes are distributed across multiple nodes rather than centralized in a single entity. The term will refer to a specific decentralized network that has a single unique identifier. Networks may be implemented as a blockchain or another distributed ledger technology. 
+
+**node**: An RPC node that facilitates interactions with a decentralized network.
+
+**Wallet API**: [The JSON-RPC interface](https://docs.metamask.io/wallet/concepts/apis/) that applications can use to programmatically interact with MetaMask wallet clients.
+
+**authorization**: A permission to use a resource. This term may be used interchangeably with the term **permission**.
+
+**scope**: A uniquely identified domain for which authorizations can be applied (see [CAIP-217](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-217.md) for further definition).
+
+**wallet scope**: A scope that applies to capabilities that a MetaMask Wallet client can handle in a relatively self-contained way. They are not specific to a network or namespace.
+
+**eip155:wallet scope**: A CAIP-2 identifier with a reserved `wallet` as the `reference`. For example "eip155:wallet" indicates a scope that is not specific to a particular Ethereum network but is related to the `eip155` `namespace`.
```

Comment:
personally prefer `wallet:eip155`, but there are several different views on this already

Reply:
one minor advantage I see with `eip155:wallet` is that it appears to be compatible with CAIP-2's `reference` [syntax](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-2.md#syntax).